environments/post_build

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
+
+
+if [ ! -d ~/.config/dask ]; then
+  mkdir -p ~/.config/dask
+fi
+
```

Comment:
Shouldn't we copy the `jobqueue.yaml` after this step?

Reply:
Let's keep it as an explicit step for now. Hard to know the right source path depending on where the script is executed from. Also good to highlight that this file exists.